apps/backend/.env.example

```diff
@@ -42,6 +42,12 @@ SLACK_APP_CLIENT_SECRET=
 # UPLOADER_HOST=
 UPLOADER_PATH_PREFIX=
 
+# See https://sailsjs.com/documentation/concepts/logging for options.
```

Comment:
The sails logging setting of "silent" is super helpful for any debug logging other than query debugging in the Backend. Highly recommended. There are defaults on both of these in their config files so nothing new or bad happens if you don't set them in your .env.